package.json

```diff
@@ -32,6 +32,7 @@
     "codecov": "1.0.0",
     "coveralls": "2.13.1",
     "istanbul": "0.4.5",
-    "mocha": "2.3.3"
+    "mocha": "^2.3.3",
```

Comment:
Ok! Sounds good, if some future import fix is implemented by one of these packages we can update it.